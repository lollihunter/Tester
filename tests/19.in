20 20 20
1 1
2 11
13 1
8 1
11 8
17 12
2 16
18 8
12 14
18 2
8 19
4 20
16 6
12 17
7 14
9 9
5 8
4 18
8 12
20 20
