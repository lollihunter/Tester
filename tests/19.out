5
1 17 19 9 20 
